### PR TITLE
Updates for the ISLE 1.4.0 release, including release page.

### PR DESCRIPTION
**Pull request by bseeger**
Many updates for the ISLE 1.4.0 Release.   Make sure the images are up before accepting this PR.

### CODE REVIEW

File: docs/release-notes/release-1-4-0.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Release Notes - ISLE v.1.3.0, 2019-09-27
```
Comment (marksandford) [inline docs/release-notes/release-1-4-0.md]: # Release Notes - ISLE v.1.4.0, 2019-11-25

File: docs/release-notes/release-1-4-0.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+* Shaun Trujillo (Mount Holyoke College), documentation review
+
+### ISLE (main project)
+
+
+* This is an ISLE security release.  Images have been updated and only minor adjustments in version happened.
+
+### Upgrade
+
+If you are coming from ISLE releases `1.3.0` - there will be minamal steps to update.
```
Comment (marksandford) [inline docs/release-notes/release-1-4-0.md]: If you are coming from ISLE release `1.3.0` or earlier - then you **need to follow the instructions** given in the [Cleanup ISLE git repository for Release v.1.4.0](https://islandora-collaboration-group.github.io/ISLE/cookbook-recipes/isle-v140-git-cleanup/) documentation.

File: docs/release-notes/release-1-4-0.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+### ISLE (main project)
+
+
+* This is an ISLE security release.  Images have been updated and only minor adjustments in version happened.
+
+### Upgrade
+
+If you are coming from ISLE releases `1.3.0` - there will be minamal steps to update. 
+If you are coming from an ISLE release older then `1.3.0`, please read the release notes for `1.3.0` to understand
+the paradigm shift that happened between previous releases.
```
Comment (marksandford) [inline docs/release-notes/release-1-4-0.md]: the paradigm shift that happened between previous releases.  You will also need to follow the instructions given in the [Cleanup ISLE git repository for Release v.1.4.0](https://islandora-collaboration-group.github.io/ISLE/cookbook-recipes/isle-v140-git-cleanup/) documentation.

File: docs/release-notes/release-1-4-0.md (hunk 2 above)
Comment (bseeger) [reply docs/release-notes/release-1-4-0.md]: Adding this in

File: docs/release-notes/release-1-4-0.md (hunk 3 above)
Comment (bseeger) [reply docs/release-notes/release-1-4-0.md]: Adding this in

File: docs/release-notes/release-1-4-0.md (hunk 1 above)
Comment (bseeger) [reply docs/release-notes/release-1-4-0.md]: Just fixed this.Thanks!

File: docs/release-notes/release-1-4-0.md (hunk 2 above)
Comment (bseeger) [reply docs/release-notes/release-1-4-0.md]: Fixed now

File: docs/release-notes/release-1-4-0.md (hunk 3 above)
Comment (bseeger) [reply docs/release-notes/release-1-4-0.md]: Fixed now

**Pull request merged**